src/content/self-hosted/administration/configuration.mdx

```diff
@@ -724,10 +724,21 @@ networkPolicies:
 ### podSecurityPolicy
 
 Decide if you want [pod security policy](https://kubernetes.io/docs/concepts/security/pod-security-policy/) to be added to the service accounts created by Okteto.
-Enabled by default.
 
-- `useAppArmor`: Add the apparmor security annotations to the Okteto pod security policies.
-  Enabled by default.
+#### enabled
+
+Whether to enable pod security policy (defaults to `true`).
+
+#### useAppArmor
+
+Add the apparmor security annotations to the Okteto pod security policies. (defaults to `true`).
+
+If `useAppArmor=true` the following labels are added to the existing pod security policy:
```

Comment:
```suggestion
If `true` the following labels are added to the existing pod security policy:
```

Reply:
Are they labels or annotations?

Reply:
Annotations, sorry, I took the full block from the other PR... Fixing...